node/blockchain/tests/test_rest_api/test_signed_change_request/test_node_declaration.py

```diff
@@ -19,10 +19,30 @@ def test_node_declaration_signed_change_request_can_be_sent_via_api(api_client,
     assert signed_change_request.signature
 
     payload = signed_change_request.dict()
-    # TODO(dmu) CRITICAL: Expect `assert response.status_code == 204` instead once `perform_create()` is implemented
-    #                     https://thenewboston.atlassian.net/browse/BC-167
-    with pytest.raises(NotImplementedError, match=f'"identifier": "{regular_node.identifier}"'):
-        api_client.post('/api/signed-change-request/', payload)
+    response = api_client.post('/api/signed-change-request/', payload)
+    assert response.status_code == 201
+    assert response.json() == payload
+
+
+@pytest.mark.django_db
+def test_node_declaration_signed_change_request_with_invalid_account_lock(
```

Comment:
Unittests for each role is needed PV, CV and regular node. You may put stubs and move out tasks, but we need them

Reply:
I added stubs and created task in Jira https://thenewboston.atlassian.net/browse/BC-188